src/post/post.service.ts

```diff
@@ -288,6 +288,7 @@ export class PostService {
         account: Account,
         content: string,
         image?: URL,
+        altText?: string,
```

Comment:
Same comment as above, I think putting the image url and alt text into an object is a good pattern, and we should use it here too